lib/interface.js

```diff
@@ -141,6 +172,37 @@ Mocha.interfaces['gagarin'] = module.exports = function (suite) {
 
 }
 
+function wrapPromisesForFiber(obj, methodList) {
+  methodList.forEach(function(method) {
+    var original = obj[method];
+    obj[method] = function() {
```

Comment:
I think proxying will be easy for now. We'll go for that now.
I'll work on this.